code/renderers/svelte/src/__test__/composeStories/portable-stories.test.ts

```diff
@@ -0,0 +1,170 @@
+/// <reference types="@testing-library/jest-dom" />;
+import { it, expect, vi, describe, afterEach } from 'vitest';
+import { render, screen, cleanup } from '@testing-library/svelte';
+// import '@testing-library/svelte/vitest';
+import { expectTypeOf } from 'expect-type';
+import type { Meta } from '../..';
+import * as stories from './Button.stories';
+// import type Button from './Button.svelte';
+import type Button from './Button.svelte';
+import { composeStories, composeStory, setProjectAnnotations } from '../../portable-stories';
+
+// example with composeStories, returns an object with all stories composed with args/decorators
+const { CSF3Primary, LoaderStory } = composeStories(stories);
+
+// example with composeStory, returns a single story composed with args/decorators
+const Secondary = composeStory(stories.CSF2Secondary, stories.default);
+describe('renders', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  // TODO args are not being passed down, now that PreviewRender is used
+  it.skip('renders primary button', () => {
+    render(CSF3Primary.Component, { ...CSF3Primary.props, label: 'Hello world' });
+    const buttonElement = screen.getByText(/Hello world/i);
+    expect(buttonElement).not.toBeNull();
+  });
+
+  it('reuses args from composed story', () => {
+    render(Secondary.Component, Secondary.props);
+    const buttonElement = screen.getByRole('button');
+    expect(buttonElement.textContent).toMatch(Secondary.args.label);
+  });
+
+  // TODO TypeError: component.$on is not a function
```

Comment:
It's gone from Svelte 5, I don't think this is Storybook related, I assume you're just reading `testing-library` docs referencing Svelte 4 features.